relay/src/healthcheck.rs

```diff
@@ -0,0 +1,51 @@
+use std::time::Duration;
+
+use anyhow::{Result, format_err};
+use clap::ArgMatches;
+use reqwest::blocking::Client;
+
+pub fn healthcheck(matches: &ArgMatches) -> Result<()> {
+    let mode = matches.get_one::<String>("mode").unwrap();
+    let timeout = matches.get_one::<u64>("timeout").unwrap();
+    let host = matches.get_one::<String>("host").unwrap();
+    let port = matches.get_one::<u16>("port").unwrap();
+
+    // Make sure `mode` is either `live` or `ready`. Otherwise, return an error.
+    if mode != "live" && mode != "ready" {
+        return Err(format_err!(
+            "Invalid mode. Expected `live` or `ready`, got `{}`.",
+            mode
+        ));
+    }
+
+    let client = Client::builder()
+        .timeout(Some(Duration::from_secs(*timeout)))
+        .build()
+        .unwrap_or_default();
+    let response = client
+        .get(format!(
+            "http://{}:{}/api/relay/healthcheck/{}/",
+            host, port, mode
+        ))
+        .send();
+
+    match response {
+        Ok(response) => {
+            if response.status().is_success() {
+                println!("Relay is healthy.");
```

Comment:
We generally avoid prints, if they are useful we can keep them, otherwise I would just only use the status code and instead work more with some log messages.